ProcessList.c

```diff
@@ -338,22 +338,37 @@ static void ProcessList_buildTreeBranch(ProcessList* this, pid_t pid, int level,
    if (pid == 0)
       return;
 
-   Vector* children = Vector_new(Class(Process), false, DEFAULT_SIZE);
-
-   int lastShown = 0;
-   for (int i = Vector_size(this->processes) - 1; i >= 0; i--) {
-      Process* process = (Process*)Vector_get(this->processes, i);
+   // The vector is sorted by parent PID, find the start of the range by bisection
+   int vsize = Vector_size(this->processes);
+   int l = 0;
+   int r = vsize;
```

Comment:
`size_t`

Reply:
The same consistency question: this index is used with Vector, and the Vector API uses `int`, not `size_t`.

Reply:
Leave the `int`/`size_t` rework for another PR …